targets/slack.js

```diff
@@ -10,3 +10,25 @@ module.exports.uploadMD = ({ title, content, channels = SLACK_CHANNEL }) => web.
   title,
   filetype: 'post',
 })
+
+module.exports.notionNotify = ({ url, title, summary }) => web.chat.postMessage({
```

Comment:
while the immediate application is based on notion integration, I feel this can be more generically named as `postSummary` (or `sendSummary`)